cmd/namespace-lock.go

```diff
@@ -147,7 +148,13 @@ func (di *distLockInstance) GetLock(timeout *dynamicTimeout) (timedOutErr error)
 	lockSource := getSource(2)
 	start := UTCNow()
 
-	if !di.rwMutex.GetLock(di.ctx, di.opsID, lockSource, timeout.Timeout()) {
+	// Lockers default to standard storage class always.
+	tolerance := globalStorageClass.GetParityForSC(storageclass.STANDARD)
```

Comment:
`tolerance` is fixed for a given storage class value. It doesn't change between two `GetLock` calls. So, it makes sense to move setting of `tolerance` to `NewDRWMutex` at the time of process start up (immediately after config is loaded).
This way the Lock/Unlock calls will remain untouched.